src/scripts/commands/dataValues.ts

```diff
@@ -168,3 +175,83 @@ const postDanglingValuesCmd = command({
         new PostDanglingValuesUseCase(dataValuesRepository, danglingDataValuesRepository).execute(args);
     },
 });
+
+const monitoringDataValues = command({
+    name: "monitoring-values",
+    description:
+        "Monitoring data values and sending an email depending on how long has passed since the last Updated",
+    args: {
+        url: getApiUrlOption(),
+        storage: option({
+            type: oneOf(["datastore", "json"]),
+            long: "storage",
+            description: "datastore or json",
+        }),
+        dataStoreNameSpace: option({
+            type: string,
+            long: "datastore-namespace",
+            defaultValue: () => "namespace to be used ",
+        }),
+        jsonSettingsPath: option({
+            type: string,
+            long: "json-settings-path",
+            description: "Path to json file that contains the dataset settings",
+            defaultValue: () => "",
+        }),
+        jsonExecutionsPath: option({
+            type: string,
+            long: "json-executions-path",
+            description: "Path to json file that contains the dataset executions information",
+            defaultValue: () => "",
+        }),
+        emailPathTemplate: option({
+            type: string,
+            long: "email-path-template",
+            description: "Path to the json file with email template information",
+        }),
+        sendEmailAfterMinutes: option({
+            type: number,
+            long: "send-email-after-minutes",
+            description: "Number of minutes to wait before sending the email notification. Default to 5",
+            defaultValue: () => 5,
```

Comment:
However, here the defaultValue makes total sense. Can you move 5 to a variable to be both used in description and defaultValue?